doc/sphinx-guides/source/installation/advanced.rst

```diff
@@ -68,43 +68,45 @@ Once a standardized version of you Custom Terms are registered as a license, an
-and preventing worker threads from becoming locked serving them. Since
-zipping is also a CPU-intensive task, it is possible to have this
-service running on a different host system, freeing the cycles on the
-main Application Server. (The system running the service needs to have
-access to the database as well as to the storage filesystem, and/or S3
-bucket).
-
-Please consult the scripts/zipdownload/README.md in the Dataverse Software 5.0+ source tree for more information. 
-
-To install: You can follow the instructions in the file above to build
-``ZipDownloadService-v1.0.0.jar``. It will also be available, pre-built as part of the Dataverse Software 5.0 release on GitHub. Copy it, together with the shell
-script scripts/zipdownload/cgi-bin/zipdownload to the cgi-bin
-directory of the chosen Apache server (/var/www/cgi-bin standard). 
-
-Make sure the shell script (zipdownload) is executable, and edit it to configure the
-database access credentials. Do note that the executable does not need
-access to the entire Dataverse installation database. A security-conscious admin
-can create a dedicated database user with access to just one table:
-``CUSTOMZIPSERVICEREQUEST``.
-
-You may need to make extra Apache configuration changes to make sure /cgi-bin/zipdownload is accessible from the outside.
-For example, if this is the same Apache that's in front of your Dataverse installation Payara instance, you will need to add another pass through statement to your configuration:
+As of Dataverse Software 5.0 we offer an **experimental** optimization for the multi-file, download-as-zip functionality.
+If this option (``:CustomZipDownloadServiceUrl``) is enabled, instead of enforcing the size limit on multi-file zipped
+downloads (as normally specified by the option ``:ZipDownloadLimit``), we attempt to serve all the files that the user
+requested (that they are authorized to download), but the request is redirected to a standalone zipper service running
+as a cgi-bin executable under Apache.
+
+Thus moving these potentially long-running jobs completely outside the Application Server (Payara); and preventing
+worker threads from becoming locked serving them. Since zipping is also a CPU-intensive task, it is possible to have
+this service running on a different host system, freeing the cycles on the main Application Server. (The system running
+the service needs to have access to the database as well as to the storage filesystem, and/or S3 bucket).
+
+Please consult the `README at scripts/zipdownload <https://github.com/IQSS/dataverse/tree/master/scripts/zipdownload>`_
+in the Dataverse Software 5.0+ source tree for more information.
+
+To install:
+
+1. Follow the instructions in the file above to build ``zipdownloader-0.0.1.jar``. (Also available from
+   `zipper.zip <https://github.com/IQSS/dataverse/releases/download/v5.0/zipper.zip>`_ of the
+   `Dataverse Software 5.0 release on GitHub <https://github.com/IQSS/dataverse/releases/tag/v5.0>`_).
```

Comment:
I would prefer not to modify this file (zipper.zip) that we released as part of 5.0. In part because we want this to work as advertised in the 5.0-5.9 versions of the guide. 
Let me think of how to modify this section. I'm thinking of mentioning that we have renamed this jar file as of this release; and that it used to be distributed under a different name; but everything should still be working the same...

Reply:
FWIW: this hasn't been part of any releases since 5.0, which is why I linked to it directly.

Reply:
I made a couple of small changes to this section.